recipe/meta.yaml

```diff
@@ -19,22 +19,25 @@ build:
 
 requirements:
   build:
-    # python is a build requirement on Windows to resolve features.
-    - python           # [win]
+    # for feature activation
+    - vc  9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
```

Comment:
Correct. This won't work. We would have to do something similar to what `tk` did in PR ( https://github.com/conda-forge/tk-feedstock/pull/2 ). However, that doesn't work either. Because `conda-build-all` doesn't have `vc` support. So we have re-renderings ripping out key functionality each time and we have to go back and fix it.

We need to have `conda-build-all` ( https://github.com/SciTools/conda-build-all/pull/36 ) and/or `conda-build` ( https://github.com/conda/conda-build/issues/1445 ) understand `vc`. Until that happens, I think we should wait on this. Especially as we have a ton of re-renderings to do in the next month to fix Travis CI, we don't need to be dodging around this issue too.